### DISCUSSION TITLE
HW_03 - getting started

### DISCUSSION THREAD

**Discussion by cooperrc** [cooperrc/engineering-dynamics#11] [General]
Greetings, 
What are the toughest concepts/problems in HW_03 so far? Have you been able to create equations of motion based upon free body diagrams?

**Comment by leena-alam** [comment]
So far, most of the concepts in HW 3 have been okay! They line up with what I have learned in past classes. A hangup for me is learning some new notations and new ways of calculating familiar topics.

**Reply by cooperrc** [reply]
Thanks for sharing, glad you see how other course concepts can be directly applied here. 

There are some notations that are a little awkward to use at first, like unit vectors and dots instead of d/dt, but it starts to be super useful as you build bigger problems or interacting systems.
